packages/react-scripts/config/webpack.config.js

```diff
@@ -240,6 +243,9 @@ module.exports = function(webpackEnv) {
               // Turned on because emoji and regex is not minified properly using default
               // https://github.com/facebook/create-react-app/issues/2488
               ascii_only: true,
+              // Added for profiling in devtools
+              keep_classnames: isEnvProductionProfile,
+              keep_fnames: isEnvProductionProfile,
```

Comment:
I think these 2 are the top-level `minify` options, not part of `output` options. Per the [doc](https://github.com/terser/terser#minify-options)

```js
{
  output: {
    // ...
  },
  keep_classnames: isEnvProductionProfile,
  keep_fnames: isEnvProductionProfile,
}
```

Reply:
oof! Thank you for the help 😄